Sources/ScipioKitCore/ResolvedModels.swift

```diff
@@ -70,7 +76,7 @@ struct ResolvedPackage: Identifiable {
     }
 }
 
-extension ResolvedPackage {
+public extension ResolvedPackage {
     var canonicalPackageLocation: CanonicalPackageLocation {
```

Comment:
If we define this property as an extension on the ScipioKit side, ScipioKitCore will no longer need to have a SwiftPM directory.